### PR TITLE
fix admonition rendering for bq

### PR DESCRIPTION
**Pull request by jbfbell**
Fix rendering of call outs / admonitions

### CODE REVIEW

**Comment by jbfbell** [comment]
* **#37730** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37730?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jbfbell and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by jbfbell** [comment]
Closes https://github.com/airbytehq/airbyte-internal-issues/issues/2901

**Review by gisripa — APPROVED** [review]

File: docs/integrations/destinations/bigquery.md [hunk 1]
```diff
@@ -80,13 +80,19 @@ https://github.com/airbytehq/airbyte/issues/3549
 4. Enter the name for the BigQuery connector.
 5. For **Project ID**, enter your
    [Google Cloud project ID](https://cloud.google.com/resource-manager/docs/creating-managing-projects#identifying_projects).
-6. For **Dataset Location**, select the location of your BigQuery dataset. :::warning You cannot
-   change the location later. :::
+6. For **Dataset Location**, select the location of your BigQuery dataset.
+
+:::warning
+You cannot change the location later. 
+:::
```
Comment (evantahler) [inline docs/integrations/destinations/bigquery.md:87]:
```suggestion
:::

```

File: docs/integrations/destinations/bigquery.md [hunk 2]
```diff
@@ -80,13 +80,19 @@ https://github.com/airbytehq/airbyte/issues/3549
...
 7. For **Default Dataset ID**, enter the BigQuery
    [Dataset ID](https://cloud.google.com/bigquery/docs/datasets#create-dataset).
 8. For **Loading Method**, select [Standard Inserts](#using-insert) or
-   [GCS Staging](#recommended-using-a-google-cloud-storage-bucket). :::tip We recommend using the
-   GCS Staging option. :::
+   [GCS Staging](#recommended-using-a-google-cloud-storage-bucket). 
+
+:::tip
+We recommend using the GCS Staging option.
+:::
```
Comment (evantahler) [inline docs/integrations/destinations/bigquery.md:95]:
```suggestion
:::

```

**Review by evantahler — APPROVED** [review]
I think adding a whole space after an admonition will prevent linters from ruining this again

**Pull request merged**